<!-- -->
<launch>

	<arg name="is_linear_interpolation" default="True"/>
	<arg name="publishes_for_steering_robot" default="False"/>
	<!-- rosrun waypoint_follower pure_pursuit -->
	<node pkg="waypoint_follower" type="pure_pursuit" name="pure_pursuit" output="screen">
		<param name="is_linear_interpolation" value="$(arg is_linear_interpolation)"/>
		<param name="publishes_for_steering_robot" value="$(arg publishes_for_steering_robot)"/>
		<param name="use_yawerr_feedback" value="false"/>
		<param name="yawerr_feedback_gain" value="0.002"/>
		<param name="feedforward_ratio" value="0.7"/>
	</node>

</launch>
